seatunnel-core/seatunnel-core-flink-sql/pom.xml

```diff
@@ -60,6 +60,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.ververica</groupId>
+            <artifactId>flink-sql-connector-mysql-cdc</artifactId>
+            <version>${flink.cdc.version}</version>
+        </dependency>
```

Comment:
`flink-sql-connector-*` dependency should be added to `seatunnel-connectors-flink-sql-dist` module